packages/suite/src/components/wallet/CoinjoinSummary/BalanceSection.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import * as routerActions from '@suite-actions/routerActions';
+import { Card, Translation } from '@suite-components';
+import { useActions, useSelector } from '@suite-hooks';
+import { Button } from '@trezor/components';
+import { CoinjoinSessionStatus } from './CoinjoinSessionStatus';
+import { FundsPrivacyBreakdown } from './FundsPrivacyBreakdown';
+
+const Container = styled(Card)`
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    align-items: center;
+    margin-bottom: 10px;
+`;
+
+const AnonymizeButton = styled(Button)`
+    justify-content: space-between;
+    width: 154px;
+    height: 46px;
+    padding: 9px 18px;
+`;
+
+export const BalanceSection = () => {
+    const { goto } = useActions({
+        goto: routerActions.goto,
+    });
+    const { account, coinjoin } = useSelector(state => ({
+        account: state.wallet.selectedAccount.account,
+        coinjoin: state.wallet.coinjoin,
+    }));
+
+    if (!account) {
```

Comment:
nit: I think account being **falsy** is impossible, since it's checked in [`Transactions`](https://github.com/trezor/trezor-suite/blob/ad8bae2d7283df4fae9e4b299ebb050a287f3811/packages/suite/src/views/wallet/transactions/index.tsx#L47). At least it makes me want to ask a question "how is it possible to get to this part without an account" :D Honestly, I'd just cast the selectedAccount `as SelectedAccountLoaded `. Or require the `SelectedAccountLoaded` in props, then TS would prompt you if you are passing an unsafe, unchecked selectedAccount. 

This is not related only to this component, I have seen that situation several times, when something is typed with a union you have to either check again or cast to a type, as you are 100% sure that this component would not even be rendered because there is a check in a parent component.

What do you think, @Nodonisko? We were discussing usage of selectors and this _**selector vs props for vars with union types**_ thing is kinda related 👀

Reply:
added to props in fcbc1400acbd07df4d7541c35a689a99b9babf0d